### ISSUE TITLE
Duplicate value on unique index on first import

### ISSUE THREAD

**Issue by jesperweber** [KevinJump/uSync8#268]
**Describe the bug**
When running the uSync import on a clean Umbraco database it fails the first time with a duplicate value on a unique index in the cmsPropertyTypeGroup table for some of my document types (see screenshot below). 
The document types will not be imported and the content using these document types will have data lose for the properties on the document types.

When running the import a second time, the failed document types from the first run, will be imported and the content will have the lost data restored.

**To Reproduce**
1. A clean Umbraco database
2. Click on 'Import' in the uSync dashboard
3. Scroll down in the import list
4. See error from the screenshot
5. Clink on 'Import' again
6. See that the failed imports from the first round is now imported

**Screenshots**
![image](https://user-images.githubusercontent.com/2575817/130461012-df6a63d9-a673-454b-ae17-cd638cf66840.png)

**About your Site (please complete the following information):**
 - Umbraco Version: 8.15.1
 - uSync Version: 8.9.7
 - Browser: Chrome

**Comment by KevinJump** [comment]
Hi,

We are just cleaning up old, issues, i hope this was something that has either been fixed for you or not happened again. 
if this is still an issue, then feel free to drop in a comment and we will look at re-opening the issue. 

♾️

_(Note: i *think* this was to do wqith tabs/groups. we did quite a bit of work post this issue, around how tabs can change to groups, accross compotions, one of the possible errors before we did this looks a lot like this, so 🤞we fixed this one)_
